### DISCUSSION TITLE
Welcome to tensorflow-onnx Discussions!

### DISCUSSION THREAD

**Discussion by guschmue** [onnx/tensorflow-onnx#1224] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by wangqiaoshi** [comment]
hello,I want to know, what is the reason for supporting only one scan output when tensorflow_onnx_mapping
